templates/misc/component.twig

```diff
@@ -56,7 +56,7 @@ Example usage:
 
   <div class="component__container">
     {% if hasTitle %}
-      <{{ component_title_level|default('h2') }} class="component__title">
+      <{{ component_title_level|default('h2') }} class="component__title" {{ use_component_title_lang_fallback ? create_attribute(({ 'lang': lang_attributes.fallback_lang })) }}>
```

Comment:
This row is missing `dir`-attribute for the fallback language. If the page is for example in arabic, this code could fail.

Reply:
Added the `dir` attribute.